src/gui/README.md

```diff
@@ -161,6 +161,33 @@ save_histogram_image
 |`-height`| height of the image in pixels, defaults to 500px. |
 |`-width`| width of the image in pixels, defaults to 500px. |
 
+### Generate animated images
+
+```tcl
+save_gif 
```

Comment:
How about save_animated_gif?  The existing save_image command can save in various formats including gif.